src/groups/mqb/mqba/mqba_domainmanager.cpp

```diff
@@ -702,6 +702,124 @@ int DomainManager::processCommand(mqbcmd::DomainsResult*        result,
             return 0;  // RETURN
         }
     }
+    else if (command.isRemoveValue()) {
+        const bsl::string& name = command.remove().domain();
+
+        DomainSp domainSp;
+
+        if (0 != locateOrCreateDomain(&domainSp, name)) {
```

Comment:
We talked about this, but just to record it down: the reason we do this here is due to the lazy loading of domains that BlazingMQ does.  We don't want to fail if a user creates a domain configuration file, happens not to connect, and then issues this admin command, or creates a domain configuration file, uses the domain, purges the domain, and restarts the broker before issuing this admin command.

Reply:
But, this will load the domain into the resolver cache, so we do need to clear that on the second pass.